src/signals/incident-management/containers/IncidentDetail/components/StatusForm/StatusForm.test.tsx

```diff
@@ -324,13 +333,13 @@ describe('signals/incident-management/containers/IncidentDetail/components/Statu
     render(renderWithContext(withDefaultTextsSelectedState))
 
     // verify that there are default texts button is visible
```

Comment:
is the description necessary. For the simple stuff maybe just skip it. This one for example is quitte descriptive

Reply:
I didn't touch the old code.